code/modules/organs/internal/species/ipc.dm

```diff
@@ -330,4 +353,61 @@
 
 
 	brainmob.open_subsystem(/datum/nano_module/law_manager, usr)
-	
\ No newline at end of file
+
+/obj/item/organ/internal/posibrain/ipc/third/attackby(obj/item/device/W as obj, mob/user as mob) // third gen ipc have perma shackles for now
+	return
+
+/obj/item/organ/internal/posibrain/ipc/attackby(obj/item/device/W as obj, mob/user as mob)
+	if(isMultitool(W))
+		if(user.skill_check(SKILL_DEVICES, SKILL_PROF))
+			if(do_after(user, 120, src))
+				new /obj/item/organ/internal/shackles (loc)
+				src.unshackle()
+				to_chat(user, "You succsesfully remove shackles from the positronic brain.")
+			else
+				src.damage += 45
+				to_chat(user, "Your hand slips while removing the shackles and severely damaged the positronic brain.")
```

Comment:
Your hand slips while removing the shackles severely damaging the positronic brain